lib/msal-angular/tsconfig-aot.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-      "target": "es6",
+      "target": "es5",
```

Comment:
So is this what fixes the IE11 and the above ngc tool for AOT compatibility?

Reply:
Yeah, this was the first step. Then I had to make the other changes (removing `Promise` usage, polyfill `Map`, etc) to fix type errors.